### PR TITLE
docs: fix format for development document

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

fix format, before this pr:

企业微信20201106-110822@2x

after format:
企业微信20201106-111236@2x

### CODE REVIEW

File: versioned_docs/version-1.0.1/development_guides/add_chaos_daemon.md [hunk 1]
```diff
@@ -268,35 +268,34 @@ Now you are all set. It's time to verify the chaos type you just created. Take t
     ```
 6. Verify your chaos. There are different logs to check to see whether your chaos works as expected:
 
+- Check the log of `chaos-controller-manager`:
```
Comment (dcalvin) [inline versioned_docs/version-1.0.1/development_guides/add_chaos_daemon.md:271]: Should be one more level of indent, under 6

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Comment by g1eny0ung** [comment]
I often use this plugin: https://github.com/DavidAnson/vscode-markdownlint to format the `*.md` files. This can help me solve a lot of troublesome formatting issues.

**Review by dcalvin — APPROVED** [review]
LGTM

**Comment by dcalvin** [comment]
> I often use this plugin: https://github.com/DavidAnson/vscode-markdownlint to format the `*.md` files. This can help me solve a lot of troublesome formatting issues.

I think we may want to enforce some of these rules to our repo

**Comment by g1eny0ung** [comment]
> > I often use this plugin: https://github.com/DavidAnson/vscode-markdownlint to format the `*.md` files. This can help me solve a lot of troublesome formatting issues.
> 
> I think we may want to enforce some of these rules to our repo

I will try to enable this in CI. This reduces a lot of effort to solve format problems.

**Pull request merged**